### DISCUSSION TITLE
How can I show the panel range in Designer?

### DISCUSSION THREAD

**Discussion by Flithor** [AvaloniaUI/Avalonia#11908] [Q&A]
I have a control be a base layout panel, but it empty by default.
In designer it's full white background.
I want to fill the panel range by different color in designer, then the developer can know the layout.

I've find there have a designer property `d:Background`, but looks it not works.
```
[Error] 41764 UpdateXamlResult error
System.Xaml.XamlException: "Unable to find emitter for node type: XamlX.Ast.XamlAstXmlDirective Line 1, position 2."
```
(Not only use `Grid`, there are have `ListBox`, `Border`, `StackPanel`, `UniformGrid` etc.)

**Comment by timunie** [comment]
check out how Grid.ShowGridLines work
